lib/node_modules/@stdlib/math/base/special/ellipj/lib/assign.js

```diff
@@ -0,0 +1,237 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var EPS = require( '@stdlib/constants/float64/eps' );
+var SQRT_EPS = require( '@stdlib/constants/float64/sqrt-eps' );
+var PI = require( '@stdlib/constants/float64/pi' );
+
+var sincos = require( '@stdlib/math/base/special/sincos' ).assign;
+var ellipk = require( '@stdlib/math/base/special/ellipk' );
+var floor = require( '@stdlib/math/base/special/floor' );
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var cosh = require( '@stdlib/math/base/special/cosh' );
+var sinh = require( '@stdlib/math/base/special/sinh' );
+var tanh = require( '@stdlib/math/base/special/tanh' );
+var atan = require( '@stdlib/math/base/special/atan' );
+var asin = require( '@stdlib/math/base/special/asin' );
+var sin = require( '@stdlib/math/base/special/sin' );
+var cos = require( '@stdlib/math/base/special/cos' );
+var abs = require( '@stdlib/math/base/special/abs' );
+
+
+// VARIABLES //
+
+var tmp4 = new Array(4);
+var tmp2 = new Array(2);
+var ca = new Array(9);
+
+
+// MAIN //
+
+/**
+* Computes the Jacobi elliptic functions sn, cn, dn, and Jacobi amplitude am.
+*
+* @private
+* @param {number} u - input value
```

Comment:
```suggestion
* @param {number} u - input value
```

This function is part of the public API.

Reply:
Ah, I copied it from here: https://github.com/stdlib-js/stdlib/blob/b259a8375210b88f56d027014399243f067b1e5f/lib/node_modules/%40stdlib/math/base/special/sincos/lib/assign.js#L77